securedrop/config.py.example

```diff
@@ -5,6 +5,8 @@ class FlaskConfig(object):
     DEBUG = False
     TESTING = False
     WTF_CSRF_ENABLED = True
+    USE_X_SENDFILE = False
```

Comment:
As commented above these are the default values for production and should therefore be `USE_X_SENDFILE = True` to be consistent. It does not matter much because it is overriden later in the file but ... let's not confuse people ;-)